src/directives/bind.js

```diff
@@ -72,7 +74,7 @@ export function handleAttributeBindingDirective(component, el, attrName, express
     }
 }
 
-function setIfChanged(el, attrName, value) {
+function setIfChanged(el, attrName, value, namespaced = false) {
```

Comment:
Great PR, I think this line slipped in from a different feature/implementation since it doesn't seem to be used.

Reply:
Good spot! This was being used in my original implementation, but it isn't actually needed.

Reply:
`namespaced` is back?